ocaml/idl/datamodel_pool.ml

```diff
@@ -1024,6 +1034,30 @@ let set_uefi_certificates =
       ]
     ~allowed_roles:_R_POOL_ADMIN ()
 
+let set_telemetry_next_schedule =
+  call ~name:"set_telemetry_next_schedule" ~lifecycle:[]
+    ~doc:
+      "Set the time after when the next round of telemetry collection will be \
+       carried out"
+    ~params:
+      [
+        (Ref _pool, "self", "The pool")
+      ; ( DateTime
+        , "value"
+        , "The time after when the next round of telemetry collection will be \
+           carried out"
+        )
+      ]
+    ~allowed_roles:_R_POOL_ADMIN ()
+
+let reset_telemetry_uuid =
+  call ~name:"reset_telemetry_uuid" ~lifecycle:[]
+    ~doc:
+      "Reset the telemetry UUID to wipe off potential correlations on the \
```

Comment:
I changed it to a clearer description:
`Assign a new UUID to telemetry data. This can be done when the correlation between the old telemetry UUID and the pool has been leaked out. The leaking will fail to ensure the telemetry data anonymous.`